app/config/ConfigDecorator.scala

```diff
@@ -223,6 +225,16 @@ class ConfigDecorator @Inject() (
     s"$dfsFrontendHost/digital-forms/form/high-income-child-benefit-tax-charge/draft/guide"
   lazy val childBenefitsCheckIfYouCanClaim          = "https://www.gov.uk/child-benefit/overview"
 
+  lazy val claimChildBenefits: String = "https://www.gov.uk/child-benefit/how-to-claim"
+
+  lazy val reportChangesChildBenefit: String = "https://www.gov.uk/report-changes-child-benefit"
+
+  lazy val viewPaymentHistory: String = "https://www.gov.uk/child-benefit/view-payment-history"
+
+  lazy val viewProofEntitlement: String = "https://www.gov.uk/child-benefit/view-proof-entitlement"
+
+  lazy val childBenefitTaxCharge: String = "https://www.gov.uk/child-benefit-tax-charge"
+
```

Comment:
We should link to the welsh version of those urls if they exist if welsh is used on our side.

Reply:
This has been taken to gov.uk for the welsh urls